Examples/ArcGISToolkitExamples/Misc/ARStatusTableViewController.swift

```diff
@@ -0,0 +1,146 @@
+        didSet {
+            DispatchQueue.main.async{ [weak self] in
+                self?.tableView.reloadData()
+            }
+        }
+    }
+    
+    /// The calculated frame rate of the `SceneView` and `ARKit` display.
+    public var frameRate: Int = 0 {
+        didSet {
+            DispatchQueue.main.async{ [weak self] in
+                self?.tableView.reloadData()
+            }
+        }
+    }
+    
+    /// The current error message.
+    public var errorMessage: String = "None" {
+        didSet {
+            DispatchQueue.main.async{ [weak self] in
+                self?.tableView.reloadData()
+            }
+        }
+    }
+
+    /// The label for the currently selected scene.
+    public var currentScene: String = "None" {
+        didSet {
+            DispatchQueue.main.async{ [weak self] in
+                self?.tableView.reloadData()
+            }
+        }
+    }
+    
+    /// The translation factor applied to the current scene.
+    public var translationFactor: Double = 1.0 {
+        didSet {
+            DispatchQueue.main.async{ [weak self] in
+                self?.tableView.reloadData()
+            }
+        }
+    }
+    
+    /// The labels for each status item.
+    private let cellLabels = ["Tracking State", "Frame Rate", "Error", "Scene", "Translation Factor"]
+    
+    /// The height of our rows.
+    private let rowHeight: CGFloat = 24.0
+    
+    override func viewDidLoad() {
+        super.viewDidLoad()
+        
+        // Set the rowHeight of our table view.
+        tableView.rowHeight = rowHeight
+        
+        // Add a blur effect behind the table view.
+        tableView.backgroundView = UIVisualEffectView(effect: UIBlurEffect(style: .regular))
+    }
+    
+    /// Calculates and returns the height of the table view based on the row height and number of rows.
+    ///
+    /// - Returns: The calculated height of the table view.
+    public func height() -> CGFloat {
+        return CGFloat(cellLabels.count) * rowHeight
+    }
+    
+    // MARK: - Table view data source
+    
+    override func numberOfSections(in tableView: UITableView) -> Int {
+        return 1
+    }
+    
+    override func tableView(_ tableView: UITableView, numberOfRowsInSection section: Int) -> Int {
+        return (section == 0) ? cellLabels.count : 0
+    }
+    
+    override func tableView(_ tableView: UITableView, cellForRowAt indexPath: IndexPath) -> UITableViewCell {
+        // Don't reuse cells, as our table is essentially static.
+        let statusCell = UITableViewCell(style: .value1, reuseIdentifier: nil)
+        statusCell.backgroundColor = .clear
+        statusCell.textLabel?.font = UIFont.systemFont(ofSize: 12.0)
+        statusCell.detailTextLabel?.font = UIFont.boldSystemFont(ofSize: 12.0)
+        statusCell.detailTextLabel?.textColor = .black
+        
+        statusCell.textLabel?.text = cellLabels[indexPath.row]
+        
+        var detailString = ""
```

Comment:
Since this property is being initialized in every case in the following `switch` statement, you don't have to initialize it here and you could make it immutable:

```swift
let detailString: String
```